### ISSUE TITLE
Telemetry explanation in the readme and website is confusing

### ISSUE THREAD

**Issue by timothykim** [VSCodium/vscodium#110]
The following sentence

> Telemetry is enabled by a build flag which we do not pass.

is very confusing and hard to parse. When you start reading, it makes you think that telemetry is enabled. So you have to get to the second clause of the sentence to learn that it's disabled. However, the second clause can cause more confusion because it describes what the build does in order to disable telemetry. The reader has to infer that not passing the flag disables telemetry.

Can't we just directly say "Telemetry is DISABLED."?

**Comment by stripedpajamas** [comment]
I agree! That sentence always frustrated me. PR welcome to update the readme, otherwise I will update it later today when I get a few minutes.

**Comment by timothykim** [comment]
Thank you for the update. The wording hasn't updated on the website though: https://vscodium.com/ (as of this posting of this comment)
